frontend/src/components/LandingPage/FeaturedCourses.jsx

```
@@ -0,0 +1,176 @@
+	const sortedCourses = sort(coursesData).desc((u) => u.rating);
+
+	return (
+		<section className="container mx-auto px-4 py-16 bg-gray-50">
+			{/*Trending courses section*/}
+			<h2 className="text-3xl font-bold text-center mb-10 text-gray-800">
+				Trending Courses
+			</h2>
+			{/* Desktop grid layout */}
+			<div className="hidden md:grid md:grid-cols-3 gap-8">
+				{coursesData.slice(0, NoOfCardsToShow).map((course) => (
+					<div className="transform hover:scale-105 hover:shadow-xl transition-all duration-300 cursor-pointer">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)